CONTRIBUTING.rst

```diff
@@ -259,8 +270,11 @@ standards <Coding Style>`_ together with a configuration file we've provided:
 
 .. code-block:: bash
 
+   # On Linux/macOS
    uncrustify --replace --no-backup -l C -c
-     $ZEPHYR_BASE/scripts/uncrustify.cfg   my_source_file.c
+     $ZEPHYR_BASE/scripts/uncrustify.cfg my_source_file.c
```

Comment:
need a backslash on the previous line to indicate a command continuation (or join the lines as you did for the Windows example).